public/javascripts/detailedResultsSlides.js

```diff
@@ -1,5 +1,61 @@
 /* eslint-disable no-unused-vars */
 
+function fetchSlideData(textHtml) {
+  fetch(
+    `/fetch-detailed-game-data?gameId=${
+      playSessionData.sessionData[slideIndex - 1].gameId
+    }`,
+    {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      data = JSON.parse(data);
+      let stores = "";
+      let metaScoreStyle = "";
+      if (data.metaScore >= 75) {
+        metaScoreStyle = "meta-green";
+      } else if (data.metaScore >= 50) {
+        metaScoreStyle = "meta-yellow";
```

Comment:
What you're defining here is not style I guess so let's change the name to class instead